tests/IntegrationHelper.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests;
+
+use App\Domain\Integrations\Integration;
+use App\Domain\Integrations\IntegrationPartnerStatus;
+use App\Domain\Integrations\IntegrationStatus;
+use App\Domain\Integrations\IntegrationType;
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+
+trait IntegrationHelper
+{
+    public function createIntegration(UuidInterface $integrationId, array $options = []): Integration
```

Comment:
This is often called `giventhereIsAnIntegration`, see https://github.com/cultuurnet/publiq-platform/blob/main/tests/Domain/Integrations/Controllers/IntegrationControllerTest.php#L951